tailwind.config.js

```diff
@@ -2,5 +2,11 @@
 module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx,mdx}"],
   plugins: [],
-  theme: {},
+  theme: {colors: {
+      black: '#1a1523',
+      gray: '#6F6E77',
+      lightGray: '#908E96',
+      tomato: '#CA3214',
```

Comment:
トマトだといまいちピンとこず...
tomatoよりorange色の方が適切かなと思ったのですがそうでもないですかね？

```suggestion
      orange: '#CA3214',
```

Reply:
tomatoを使ったのはFigmaでその色をTomatoとしていたからです。個人的にorangeには抵抗があります。（しかし、色をどのように感じるかはかなりその方の文化的な背景が影響するので、面白いですね。）

Reply:
なるほど。確かにtomato